.gitignore

```diff
@@ -6,6 +6,7 @@ Pipfile
 Pipfile.lock
 .mypy_cache/
 .idea/
+.vscode/
```

Comment:
this should not be added to gitignore because it stores the configuration all developers should follow.

please remove it.

Reply:
I understand the importance of consistent configurations across different environments.

But I think keeping the `.vscode` folder in the repository can lead to 2 potential issues:

1. The .vscode folder often contains individual developer settings, which might conflict with personal preferences or setups.
2. Sometimes, `.vscode/*` files may include sensitive information specific to a developer's local environment.

To achieve consistency, it is possible to save essential settings in `.vscode/settings.json` while ignoring other files from this folder. 

Let me know your thoughts!

Reply:
current .vscode in the source code does not have any sensitive information. still thinking it is important to have it.